proto-src/psdbconnect/v1/connect.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
```

Comment:
This is currently a copy/paste from v1alpha1, so will likely get some changes before we implement.